distribution/docs/src/main/resources/_contents/_nsili/managing-nsili-contents.adoc

```diff
@@ -0,0 +1,253 @@
+
+The NSILI Application provides a bidirectional interface to STANAG-4559 Edition 3 compliant systems.
+
+=== Prerequisites for ${alliance-nsili}
+
+The following apps must be installed with/before ${alliance-nsili}:
+
+* <<Managing ${ddf-spatial}, ${ddf-spatial}>>
+
+=== Installing ${alliance-nsili}
+
+This app is installed by default with standard installation.
```

Comment:
Change "This app..." to "The Alliance NSILI app..." to have the text independent of the heading.